Backend/dev/migrations/rider-app/1134-add-status-column-in-driver-offer.sql

```diff
@@ -0,0 +1,9 @@
+ALTER TABLE atlas_app.driver_offer
+    ADD COLUMN status character varying(255),
+    ADD COLUMN updated_at timestamp with time zone;
+
+UPDATE atlas_app.driver_offer SET status = 'INACTIVE';
```

Comment:
Don't set here, just add default as `ACTIVE` and CURRENT_TIMESTAMP;

Reply:
for old driver offers their status will be active then which shouldn't be

Reply:
Let it be, we can't update old data that is too big.